cmd/compute-domain-daemon/main.go

```diff
@@ -17,30 +17,31 @@
 package main
 
 import (
+	"bytes"
 	"context"
 	"fmt"
 	"os"
 	"os/exec"
 	"os/signal"
 	"path/filepath"
-	"strings"
 	"sync"
 	"syscall"
+	"text/template"
 
 	"k8s.io/klog/v2"
 
-	"github.com/Masterminds/semver"
```

Comment:
The masterminds. :brain: I thought you said we only allow google and k8s pkgs. I think we should allow ourselves to make exceptions to such rule, and I am glad to see that we did that in the past.